### PR TITLE
Names of Symbols

### PR DESCRIPTION
**Pull request by kckennylau**

### CODE REVIEW

File: templates/contribute/naming.md [hunk 1]
```diff
@@ -1,6 +1,51 @@
...
+- `0`: zero
+- `+`: add
+- `-`: neg (unary) / sub (binary)
+- `1`: one
+- `*`: mul
+- `^`: pow
+- `/`: div
+- `•` (`\bu`): is smul
+- `⁻¹` (`\-1`): is inv
+- `∣` (`\|`): is dvd
```
Comment (jalex-stark) [inline templates/contribute/naming.md:39]: Why "is smul", "is inv", "is dvd" instead of "smul", "inv", "dvd"?

Comment (kckennylau) [reply templates/contribute/naming.md]: fixed

Comment (bryangingechen) [reply templates/contribute/naming.md]: Could you put all these names in between backticks as well? e.g. `or`, `and`, `of`, etc.

Comment (kckennylau) [reply templates/contribute/naming.md]: done

File: templates/contribute/naming.md [hunk 2]
```diff
@@ -1,6 +1,63 @@
...
+| `0`    |          | `zero`        |                                                             |
+| `+`    |          | `add`         |                                                             |
+| `-`    |          | `neg` / `sub` | `neg` for the unary function, `sub` for the binary function |
+| `1`    |          | `one`         |                                                             |
+| `*`    |          | `mul`         |                                                             |
+| `^`    |          | `pow`         |                                                             |
+| `/`    |          | `div`         |                                                             |
+| `•`    | `\bu`    | `smul`        |                                                             |
+| `⁻¹`   | `\-1`    | `inv`         |                                                             |
+| `∣`    | `\|`     | `dvd`         |                                                             |
```
Comment (jcommelin) [inline templates/contribute/naming.md:48]: Maybe add `\sum` and `\prod` here? Or are they already too specialised?

Comment (kckennylau) [reply templates/contribute/naming.md:48]: done

**Comment by jcommelin** [comment]
LGTM

**Pull request merged**